diff --git a/ports/unix/Makefile b/ports/unix/Makefile
index d5bd6d409..7366a9c16 100644
--- a/ports/unix/Makefile
+++ b/ports/unix/Makefile
@@ -44,7 +44,7 @@ INC +=  -I$(TOP)
 INC += -I$(BUILD)
 
 # compiler settings
-CWARN = -Wall -Werror
+CWARN = -Wall # -Werror
 CWARN += -Wextra -Wno-unused-parameter -Wpointer-arith -Wdouble-promotion -Wfloat-conversion
 CFLAGS += $(INC) $(CWARN) -std=gnu99 -DUNIX $(COPT) -I$(VARIANT_DIR) $(CFLAGS_EXTRA)
 
